src/request.ts

```diff
@@ -14,6 +15,7 @@ class Request {
   private req: AxiosInstance;
   private _logger?: Logger;
   private _debug: boolean = false;
+  private retryAfterTimeout: number = 1000;
```

Comment:
I don't think we need this variable anymore now. As we're not really waiting for anything, so we don't need to set this value to zero in the tests. So I suggest we remove it and just use the value in the place where it's useful.